lib/wificom/mqtt.py

```diff
@@ -40,16 +41,29 @@ def connect_to_mqtt(mqtt_client):
 	_mqtt_client.on_unsubscribe = unsubscribe
 
 	# Connect to MQTT Broker
-	print("Connecting to MQTT Broker...")
-	# _io.connect()
-	_mqtt_client.connect()
+	attempt = 0
+	while attempt < 3:
+		try:
+			print(f"Connecting to MQTT Broker (attempt {attempt+1})...")
+			_mqtt_client.connect()
+
+			return _mqtt_client
+		# pylint: disable=bare-except
+		except:
+			attempt += 1
+			time.sleep(1)
+	# pylint: disable=useless-else-on-loop
+	else:
+		print("Unable to connect to MQTT Broker after 3 attempts.")
+		return None
 
 	# Use _mqtt_client to subscribe to the mqtt_topic_input feed
 	_mqtt_client.subscribe(_mqtt_topic_input)
```

Comment:
Looks like we don't arrive here? while/else only makes sense if there's a `break`

Reply:
Did a small refactor to this, should be cleaner now